openbb_platform/providers/alpha_vantage/openbb_alpha_vantage/models/equity_historical.py

```diff
@@ -28,7 +28,10 @@ class AVEquityHistoricalQueryParams(EquityHistoricalQueryParams):
     Source: https://www.alphavantage.co/documentation/#time-series-data
     """
 
-    __json_schema_extra__ = {"symbol": {"multiple_items_allowed": True}}
+    __json_schema_extra__ = {
```

Comment:
could this be done at the `Field` level instead of model? the `OpenBBField` is expecting choices to be there - this would then influence the CLI

![image](https://github.com/user-attachments/assets/f6164c02-40c5-49a8-b488-dced676d3fd2)

Reply:
No, for things to show up in `openapi.json` it must be defined in `__json_schema_extra__`. Not sure why, but that is the way it appears to work.

Reply:
apparently is fine since it's allso available on the reference.json